subjects/index.js

```diff
@@ -2,7 +2,7 @@ const pg = require('pg');
 const performQuery = require('../helpers/performQuery').default
 
 async function executeSQL(context, params) {
-    const connectionString = 'postgres://postgres:postgres@localhost:5432/sample_db'
+    const connectionString = process.env['PG_DB_CONNECTION_STRING']

```

Comment:
Might be worth adding a fallback for local testing so folks don't have to setup the env var
```suggestion
    const connectionString = process.env['PG_DB_CONNECTION_STRING'] || 'postgres://postgres:postgres@localhost:5432/sample_db'

```

Reply:
Good idea 👍